packages/trader/src/App/Containers/__tests__/trade-footer-extensions.spec.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import TraderProviders from '../../../trader-providers';
+import TradeFooterExtensions from '../trade-footer-extensions';
+import { mockStore } from '@deriv/stores';
+import { RouteComponentProps, Router } from 'react-router-dom';
+import { MemoryHistory, createMemoryHistory } from 'history';
+
+describe('<TradeFooterExtensions>', () => {
+    let mock_store: ReturnType<typeof mockStore>,
+        router_prop: Partial<RouteComponentProps>,
+        customHistory: MemoryHistory;
+    beforeEach(() => {
+        mock_store = {
+            ...mockStore({
+                client: {
+                    is_logged_in: false,
+                },
+                portfolio: {
+                    active_positions_count: 0,
+                },
+                ui: {
+                    is_positions_drawer_on: false,
+                    populateFooterExtensions: jest.fn(item => item[0]),
+                },
+            }),
+        };
+        customHistory = createMemoryHistory({ initialEntries: ['/test'] });
+    });
+    const renderTraderFooterExtensions = (props: Partial<RouteComponentProps>) => {
+        customHistory = createMemoryHistory({ initialEntries: [props.location.pathname] });
+        return render(
+            <TraderProviders store={mock_store}>
+                <Router history={customHistory}>
+                    <TradeFooterExtensions {...props} />
+                </Router>
+            </TraderProviders>
+        );
+    };
```

Comment:
```suggestion
    const history = createMemoryHistory();
    const renderTraderFooterExtensions = () => {
        return render(<TraderProviders store={mock_store}>
                <Router history={history}>
                    <TradeFooterExtensions {...props} />
                </Router>
            </TraderProviders>);
    };
```
And in test cases below you would simply do:
```
history.location.pathname = '/';
renderTraderFooterExtensions();
```
will this work? :)

Reply:
this wouldn't work